articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/types-of-styling-in-css/
+title: Types of CSS (Cascading Style Sheets)
+description: This article will go over the different styling options in CSS, how to apply them, and when best to implement each style.
+author:
+date: 2020-11-02T00:00:00-13:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/types-of-styling-in-css/hero.jpg
+    alt: Types of CSS (Cascading Style Sheets) example image
+
+---
+We will be going over Cascading Style Sheets commonly known as CSS. CSS is what turns your everyday markup language, HTML, into a one of a kind expression that leaves people in admiration. CSS complements the elements in HTML to help come up with various designs that make a website pop. From background colors, font-size, font-weight, font-family, and much more the design of a website becomes limited by only your imagination.
+<!--more-->
+CSS is the main playground for any developer. To use an analogy, CSS is like all the spices you add to your food to give it a rich, enticing aroma and the desired taste when you finally present it. However, just throwing things in there randomly doesn't guarantee greatness.
```

Comment:
I like this intro - its a nice metaphor - but if you could elaborate what CSS is how it works how it ties into other programming languages used in web development.